container-registry/vulnerability-notification-function/mvnw.cmd

```diff
@@ -0,0 +1,182 @@
+@REM ----------------------------------------------------------------------------
```

Comment:
skip this

Reply:
this is standard maven wrapper.

Reply:
Please delete the maven wrapper - we don't use them in our samples